res/controllers/midi-components-0.0.js

```diff
@@ -318,21 +318,68 @@
         this.firstValueReceived = false;
     };
     Pot.prototype = new Component({
+        relative: false,
         inValueScale: function (value) { return value / this.max; },
-        input: function (channel, control, value, status, group) {
-            this.inSetParameter(this.inValueScale(value));
-            if (! this.firstValueReceived) {
-                this.firstValueReceived = true;
-                this.connect();
+        shift: function () {
+            if (this.relative) {
+                this.input = function (channel, control, value, status, group) {
```

Comment:
When holding shift in relative mode, the `input` function only stores the position of the potentiometer but does not set the value of any ControlObject. Without a way to move the potentiometer like this, there wouldn't be a point to having a relative mode.